README.md

```diff
@@ -106,6 +110,7 @@ sequences:
 nmap <unique> <leader>pe <Plug>(PickerEdit)
 nmap <unique> <leader>ps <Plug>(PickerSplit)
 nmap <unique> <leader>pt <Plug>(PickerTabedit)
+nmap <unique> <leader>pd <Plug>(PickerTabdrop)
```

Comment:
This PR has prompted me to think about it, and `tab drop` is what I want much more often than `tabedit`. What do you think of suggesting `<leader>pt` maps to `PickerTabdrop` rather than `PickerTabedit`, and leaving the latter unbound by default?

Reply:
You may want to create a buffer in another tab with same file. And this is how Vim recommends tabs' use:
tabs > windows > buffers.

Using `tab drop` is usefull for someone who uses one tab one buffer.

I would say let others decide how they want to open tabs.

I would prefer not to left something unbound that works for someone.